### PR TITLE
Update winit to 0.29.2

### LINKED ISSUE #2453: Blocking paste freezing Alacritty
# Issue
I'm on Gentoo, running Sway (and thus Wayland.) The clipboard has never really worked on Wayland, it's consistently a mess. Talks with @chrisduerr and @cmr suggested that it should be fixed on master.

Today I compiled master with rustc 1.34.2 to try, and it seems like it is still not working.

# How to reproduce
1. Running Sway (might work on other Wayland WMs?)
2. Open Firefox with `GDK_BACKEND=wayland` set.
3. Select and copy some text on Firefox.
4. Switch to another workspace where Alacritty is.
5. Paste text (ctrl+shift+v)

The result should be a freeze and eventually a crash.

On a release build I get this:
```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: EventsLoopClosed', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

On a debug build with backtracing I get this:
```
alacritty on  master via 𝗥 v1.34.2 [I] ➜ RUST_BACKTRACE=1 cr
   Compiling alacritty v0.3.2 (/home/bemeurer/src/alacritty/alacritty)
    Finished dev [unoptimized + debuginfo] target(s) in 7.18s
     Running `target/debug/alacritty`
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: EventsLoopClosed', src/libcore/result.rs:997:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:70
             at src/libstd/sys_common/backtrace.rs:58
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:200
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:215
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:478
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:385
   6: rust_begin_unwind
             at src/libstd/panicking.rs:312
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::result::unwrap_failed
             at /var/tmp/portage/dev-lang/rust-1.34.2/work/rustc-1.34.2-src/src/libcore/macros.rs:17
   9: <core::result::Result<T, E>>::unwrap
             at /var/tmp/portage/dev-lang/rust-1.34.2/work/rustc-1.34.2-src/src/libcore/result.rs:798
  10: winit::platform::platform::wayland::keyboard::init_keyboard::{{closure}}
             at /home/bemeurer/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.19.1/src/platform/linux/wayland/keyboard.rs:111
  11: smithay_client_toolkit::keyboard::implement_kbd::{{closure}}::{{closure}}::{{closure}}
             at /home/bemeurer/.cargo/registry/src/github.com-1ecc6299db9ec823/smithay-client-toolkit-0.4.5/src/keyboard/mod.rs:720


^C
```

### LINKED ISSUE #3108: High wakeup count on idle
Alacritty 0.4.0 on Wayland/Sway

Using Powertop, I see Alacritty having a steady ~19-20 wakeups/s. This is on a **completely idle** prompt, with the default (no file) config.

There's no measurable CPU usage, just the high wakeup count. This does however cause battery drain.

Just for reference, XTerm shows 0 wakeups/s in the same setup.

Not very high-priority I suppose :)

### LINKED ISSUE #3601: Xorg Clipboard paste's content is "lagged behind"
After a day or two of Alacritty instance uptime, I notice that clipboard updates (usually copies) from other programs get "lagged behind". When performing a paste action, the old (i.e. wrong) content gets emitted. I have to wait a second between a copy from one program, until I paste in Alacritty. A new Alacritty instance being spawn does not exhibit this behavior, and sees updated clipboard content without a lag.

This might be a copypaste issue or a x11-clipboard issue, but I'm not sure how to debug this.

### System

OS: Linux
Version: alacritty 0.4.2 (f68de375b67d)
Linux/BSD: X11

### LINKED ISSUE #6848: Truncated text on copy to clipboard
### Issue Description
Copy selection gets truncated to about 65k chars.

I'm not that experienced with rust but poked a little around and maybe this is useful:
- The final call where the text gets cut occures somewhere around here: https://github.com/alacritty/alacritty/blob/d40198da53f4c1c882901e364aca59e9b2ef2367/alacritty/src/clipboard.rs#L73
- I've added a debug log message right before that to log the length of `text`, which is actually correct.
- I've replaced `text` with a temporary variable initialized with about 2 MiB of text from a file using `include_str!("clip.txt")`. The length is correct but after the text gets passed into `clipboard.set_contents()` it's truncated again
- I've created a minimal test app using the example code and the same `include_str!("clip.txt")` of [copypasta/examples/hello_world.rs](https://github.com/alacritty/copypasta/blob/94e44c83dbaf08582309b297f0ae3b1b8cd1b17c/examples/hello_world.rs) and that does actually works as expected, the clipboard is not truncated.

### Steps to reproduce
- Build & Run
- Generate output larger then about 65k chars, I used `journalctl --since today --no-pager`
- Select all
- Copy to clipboard using default keybinding `Ctrl+Shift+C`
- Paste clipboard into other application, i.e. VS Code or gedit

### System

OS: Linux - Fedora 37 Workstation (Gnome3 Wayland)

Version: alacritty 0.12.0 (via `cargo install alacritty`) and alacritty 0.13.0-dev (commit d40198da53f4c1c882901e364aca59e9b2ef2367)

### Logs

Crashes: N/A

Config: [alacritty.txt](https://github.com/alacritty/alacritty/files/11184617/alacritty.txt)

Font/Terminal size: `alacritty -vv` -> [Alacritty.log](https://github.com/alacritty/alacritty/files/11184609/Alacritty.log)

Keyboard and bindings: `alacritty --print-events` -> [Alacritty.log](https://github.com/alacritty/alacritty/files/11184607/Alacritty.log)

### LINKED ISSUE #7201: Delayed prompt display in Alacritty when using touchpad
### System

- **OS:** Linux Mint 21.2 x86_64  
- **Version:** alacritty 0.13.0-dev (8eed1722)  
- **Linux/BSD:** X11  

### Logs

[events.log](https://github.com/alacritty/alacritty/files/12545993/events.log)

**Font/Terminal size:**  
- Window scale factor: 2
- Cell size: 18 x 36
- Padding: 0 x 0
- Width: 1440, Height: 1080
- PTY dimensions: 30 x 80

**Keyboard and bindings:** The `events.log` has been created and can be attached if required.

### Issue Description

**Hardware:** Host: MacBookPro11,3 1.0

When launching Alacritty, the prompt only appears immediately when the cursor is inside the Alacritty window. This behavior seems to occur only when the touchpad has been used. If a mouse is used, this behavior does not occur.

### LINKED ISSUE #7236: Bell causes window rendering updates to stop on Windows
When I do something in WSL that causes a terminal bell, the Alacritty window ceases to update until I move the window to another monitor, resize it, etc.

I bisected the issue to this commit: https://github.com/alacritty/alacritty/commit/a189861880aafcce760e3a142bc2d391f2801f64

### System

OS: Windows
Version: master

### PR DESCRIPTION
**Pull request by kchibisov**
Fixes #7236.
Fixes #7201.
Fixes #6848.
Fixes #3601.
Fixes #3108.
Fixes #2453.

### CODE REVIEW

**Comment by sukima** [comment]
I would like to confirm, after applying this patch that…

* My slow refreshes while coding in Vim are **gone**
* My TMUX is responding way **faster**
* My Emojis are crystal clear now
* My fonts are much bigger (my eyes thank you)
* Everything runs so much faster and snappier

Thank you!!!!

**Comment by kchibisov** [comment]
@sukima I'm not sure why you have so many things fixed if this PR literally changed nothing compared to them. Probably an oversight.

**Comment by sukima** [comment]
It is possible that a `git pull` also contributed to several fixes. But this PR defiantly fixed the #7201 problem I've been having.

**Comment by proski** [comment]
This PR solves major issues with MacOS 14 Sonoma that was released today. Alacritty 0.12.2 doesn't work at all, it hangs. The current master branch makes the first line of text appear partially behind the title. Selecting text with the mouse selects on the line above the expected one. Alacritty build from this PR is working properly for me.

Not sure if we need a bug to track Sonoma issues, but I believe a new release is needed very soon.

**Comment by kchibisov** [comment]
did mac broke things **so** badly? We have the same macOS code for 4 years, I'm pretty sure we're not the only one affected by that.

**Comment by proski** [comment]
Everything else appears to be working correctly, including the standard Terminal app. I've opened a ticket #7242 specifically for that issue.

**Comment by kchibisov** [comment]
Hm, I think it's because Apple changed the way resizes work and the way viewport sizes are being reported, and given that winit used different thing it sort of broke stuff.

Out of curiousity, could check with `--print-events` resizes on the `0.12.2`? Maybe that's this issue https://github.com/rust-windowing/winit/issues/2876 ? 

I question, because maybe we should really back port and do a patch release of the winit itself, and maybe even alacritty?

**Comment by proski** [comment]
I'm sure it's that issue.
```
[0.000274625s] [INFO ] [alacritty] Welcome to Alacritty
[0.000515667s] [INFO ] [alacritty] Version 0.12.2 (9d9982df)
[0.001379125s] [INFO ] [alacritty_config_derive] No config file found; using default
[0.016426167s] [INFO ] [alacritty] winit event: NewEvents(Init)
[0.016482667s] [INFO ] [alacritty] winit event: Resumed
[0.016543250s] [INFO ] [alacritty] Using Apple CGL
[0.042976333s] [INFO ] [alacritty] Window scale factor: 2
[0.071208917s] [INFO ] [alacritty] Running on Apple M2 Max
[0.071232042s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.082526708s] [INFO ] [alacritty] Cell size: 13 x 25
[0.082547708s] [INFO ] [alacritty] Padding: 0 x 0
[0.082559708s] [INFO ] [alacritty] Width: 1600, Height: 1200
[0.086401458s] [INFO ] [alacritty] PTY dimensions: 48 x 123
[0.088278708s] [INFO ] [alacritty] Initialisation complete
[0.092647333s] [INFO ] [alacritty] winit event: RedrawRequested(WindowId(WindowId(5636403584)))
[0.095163458s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(5636403584)), event: Resized(PhysicalSize { width: 4294967295, height: 4294967295 }) }
[0.095196583s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(5636403584)), event: Focused(false) }
[0.095216458s] [INFO ] [alacritty] winit event: MainEventsCleared
[0.095234250s] [INFO ] [alacritty] winit event: RedrawRequested(WindowId(WindowId(5636403584)))
[0.095249292s] [INFO ] [alacritty] winit event: RedrawEventsCleared
thread 'PTY reader' panicked at alacritty_terminal/src/tty/unix.rs:418:25:
attempt to multiply with overflow
```

**Comment by kchibisov** [comment]
Thanks, I'll try to see whether we can backport that, but maybe not into alacritty itself.

At least the rest of the ecosystem could have the fix without waiting on the update of winit.

**Comment by proski** [comment]
I reverted https://github.com/rust-windowing/winit/issues/2876 on a local copy of winit and used it with the code from this PR. It triggered **both** issues I reported in #7242, the overflow panic in the Windowed mode and the lines behind the title in the Maximized mode. So apparently the backport would be very useful, considering that users are updating to MacOS 14 en masse after the release.

**Comment by proski** [comment]
Please don't forget to update copypasta version in the commit title.

**Comment by tbozan** [comment]
With this last version of pr only upper left quarter of a terminal window is clickable (beam cursor, can select text, instead of regular cursor and unable to select text in other three quarters).
As i'm using 2x scaling i'm guessing there's a chance that it has something to do with hidpi.
Last working version of this pr was 67382eb.

**Comment by kchibisov** [comment]
@tbozan could you be more specific on that matter?

**Comment by tbozan** [comment]
Sorry about that. Latest mesa and linux kernel, picom compositor.
Basically when i'm interacting with the mouse (clicking, selecting text...) with upper left quarter of the window everything works as expected. When interacting with other three quarters nothing happens, like i'm clicking on a screenshot of a window.

**Comment by kchibisov** [comment]
@tbozan could you provide the `--print-events` log and the log when you click on different parts of the window?

**Comment by tbozan** [comment]
When moving the cursor or clicking on upper left quarter i'm getting the usual events:
```
[2.855835398s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 743574320 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.855840097s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: PhysicalPosition { x: 780.0, y: 292.0 } } }
[2.855844487s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 0, value: 782.0 } }
[2.855849327s] [INFO ] [alacritty] winit event: AboutToWait
[2.856835703s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 744574296 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.856840923s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: PhysicalPosition { x: 784.0, y: 292.0 } } }
[2.856845193s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 0, value: 786.0 } }
[2.856849963s] [INFO ] [alacritty] winit event: AboutToWait
[2.857835939s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 745519402 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.857842199s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: PhysicalPosition { x: 791.0, y: 293.0 } } }
[2.857849399s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 0, value: 793.0 } }
[2.857853569s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 1, value: 371.0 } }
[2.857858689s] [INFO ] [alacritty] winit event: AboutToWait
[2.858838005s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 746585238 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.858843235s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: PhysicalPosition { x: 795.0, y: 293.0 } } }
[2.858847995s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 0, value: 797.0 } }
[2.858853195s] [INFO ] [alacritty] winit event: AboutToWait
[2.859838050s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 747585753 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.859844100s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: PhysicalPosition { x: 799.0, y: 293.0 } } }
[2.859848860s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: AxisMotion { device_id: DeviceId(X(DeviceId(2))), axis: 0, value: 801.0 } }
[2.859854170s] [INFO ] [alacritty] winit event: AboutToWait
[2.860839336s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18595, tv_nsec: 748579459 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 132330815 }) })
[2.860844486s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: CursorLeft { device_id: DeviceId(X(DeviceId(2))) } }
[2.860849576s] [INFO ] [alacritty] winit event: AboutToWait
[3.243674390s] [INFO ] [alacritty] winit event: NewEvents(ResumeTimeReached { start: Instant { tv_sec: 18595, tv_nsec: 749519005 }, requested_resume: Instant { tv_sec: 18596, tv_nsec: 132330815 } })
[3.243689990s] [INFO ] [alacritty] winit event: AboutToWait
[3.243698980s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 132362189 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.243704300s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(25165827))), payload: BlinkCursor })
[3.243709420s] [INFO ] [alacritty] winit event: AboutToWait
[3.243714580s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 132378899 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.243720290s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(25165827)), event: RedrawRequested }
[3.244190609s] [INFO ] [alacritty] winit event: AboutToWait
[3.244201948s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 132864457 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.244207888s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(25165827))), payload: Frame { force: false } })
```

With other three quarters nothing happens when moving the cursor, only when i click on the window i get:
```
[3.314887544s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 132881467 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.314906624s] [INFO ] [alacritty] winit event: AboutToWait
[3.398846904s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 203576913 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.398871134s] [INFO ] [alacritty] winit event: AboutToWait
[3.668881109s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 287541613 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.668898769s] [INFO ] [alacritty] winit event: AboutToWait
[3.752828639s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 18596, tv_nsec: 557568778 }, requested_resume: Some(Instant { tv_sec: 18596, tv_nsec: 757360509 }) })
[3.752840928s] [INFO ] [alacritty] winit event: AboutToWait
[3.868705206s] [INFO ] [alacritty] winit event: NewEvents(ResumeTimeReached { start: Instant { tv_sec: 18596, tv_nsec: 641510797 }, requested_resume: Instant { tv_sec: 18596, tv_nsec: 757360509 } })
[3.868721536s] [INFO ] [alacritty] winit event: AboutToWait
```
One pair of `NewEvents(WaitCancelled...` and `AboutToWait` for click-down and click-up.

**Comment by kchibisov** [comment]
Could you also paste the log `alacritty -vv --print-events` without actually truncating it?

**Comment by tbozan** [comment]
Here it is: https://gist.github.com/tbozan/9fbe3cb28596d139bca0d6ad86b8c3ea

**Comment by kchibisov** [comment]
Does it happen with `alacritty --config-file /dev/null`?

**Comment by tbozan** [comment]
Good catch. When i comment out window.dimensions and restart alacritty everything works, even when i manually resize window.

**Comment by kchibisov** [comment]
@tbozan your issue should be fixed now. Thanks.

File: alacritty/src/event.rs [hunk 1]
```diff
@@ -1578,14 +1583,34 @@ impl Processor {
...
                     info!("Initialisation complete");
                 },
+                WinitEvent::LoopExiting => {
+                    match self.gl_display.take() {
+                        #[cfg(not(target_os = "macos"))]
+                        Some(glutin::display::Display::Egl(display)) => {
+                            // Ensure that all the windows are dropped, so the destructors for
+                            // Renderer and contexts ran.
+                            let windows = mem::take(&mut self.windows);
+                            drop(windows);
```
Comment (chrisduerr) [inline alacritty/src/event.rs:1602]: Doesn't just `self.windows.clear()` have the same effect?

File: alacritty/src/event.rs [hunk 2]
```diff
@@ -1578,14 +1583,34 @@ impl Processor {
...
+
+                            // SAFETY: the display is being destroyed after destroying all the
+                            // windows, thus no attempt to access the
+                            // EGL state will be made.
```
Comment (chrisduerr) [inline alacritty/src/event.rs:1606]: Formatting.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: alacritty/Cargo.toml [hunk 3]
```diff
@@ -26,21 +26,22 @@ version = "0.1.2-dev"
...
 once_cell = "1.12"
 parking_lot = "0.12.0"
+raw-window-handle = "0.5"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 serde_yaml = "0.9.25"
 toml = "0.8.2"
 unicode-width = "0.1"
-winit = { version = "0.29.1-beta", default-features = false, features = ["serde"] }
+winit = { version = "0.29.2", default-features = false, features = ["rwh_05", "serde"] }
```
[the remark this answers is not in the document]
Comment (kchibisov) [reply alacritty/Cargo.toml:44]: yes. probably, but we have a lot of them and you need multiple at the same time, so you won't have a giant name. You'd also have to alias `raw-window-handle` to something in your code if you use multiple handles, they are not mutually exclusive, but rather additive.

**Review by chrisduerr — APPROVED** [review]

**Comment by ProNoob135** [comment]
After this commit, Alacritty hangs if moved off my primary monitor.
`./target/debug/alacritty -vv --print-events --config-file /dev/null`
[Alacritty-3416.log](https://github.com/alacritty/alacritty/files/14042765/Alacritty-3416.log)
Here's how far it gets if launched on my other monitors:
[Alacritty-4406.log](https://github.com/alacritty/alacritty/files/14042830/Alacritty-4406.log)
I'm assuming something is off with my system configuration, but I've no idea what.
XFCE4(X11), three monitors, here's my xorg.conf:
```Section	"Monitor"
	Identifier	"HDMI-A-0"
	Option	"Position"	"0 180"
EndSection

Section	"Monitor"
	Identifier	"DisplayPort-1"
	Modeline	"2560x1440_143.97"  592.25  2560 2581 2613 2666  1440 1443 1448 1543 +hsync -vsync
	Option	"PreferredMode"	"2560x1440_143.97"
	Option	"Position"	"1920 0"
	Option	"Primary"	"true"
EndSection

Section "Monitor"
	Identifier	"DisplayPort-2"
	Option	"PreferredMode"	"1920x1080"
	Option	"Position"	"4480 360"
	Option	"DPMS"	"false"
EndSection```

**Pull request merged**